backend/de.metas.serviceprovider/src/main/java-gen/de/metas/serviceprovider/model/I_S_TimeBooking.java

```diff
@@ -82,7 +82,7 @@
 	/**
 	 * Set Booked date.
 	 *
-	 * <br>Type: DateTime
+	 * <br>Type: Date
```

Comment:
can we still have multiple bookings per day? or don't we (currently) need/want this?

Reply:
we can, we dropped the `unique` constraint when we moved from the `exportTime` endpoint to the `getTeamBookedTime`  one ( the one that provides also ids for the time records ) .